dist/licenses.txt

```diff
@@ -12,6 +12,15 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 
 @actions/github
 MIT
+The MIT License (MIT)
```

Comment:
What generates this? Afaik you're under no obligation to do this, except for gpl sources possibly.

Reply:
The build script (`ncc build index.ts --license licenses.txt`) dumps all of that. I saw it in another example, or elsewhere in some docs and thought I might as well? It's part of the "bundled" output, so it seems like including licenses there is a good idea, like any other shipped product.

Reply:
It's cool!